muikku-calendar/src/main/resources/META-INF/resources/scripts/gui/calendar.js

```diff
@@ -65,8 +65,8 @@
          values.start = eventStart;
          values.end = eventStop;
          values.status = 'CONFIRMED';
-         values.startTimeZone = 'GMT+3';
-         values.endTimeZone = 'GMT+3';        
+         values.startTimeZone = 'GMT+2';
+         values.endTimeZone = 'GMT+2';        
```

Comment:
These should most likely be "localized" too as this change i believe is based on daylight savings. More concretely, this setting should come from the users' current timezone.